### ISSUE TITLE
Can't open after the update on June 18

### ISSUE THREAD

**Issue by chikinrais** [tribalfs/Pixels#14]
**Describe the bug**
Same as the title, can't open after the update.
The app will show for a moment, and then close automatically.
Restarting the phone, reinstalling the app, and resetting the adb settings did not solve the problem.
Before updating this version, everything was fine.
Hope someone can provide a solution.

**Smartphone (please complete the following information):**
 - Device: Oneplus 9R
 - OS: Android11 ColorOS V11.2

**Are you able to complete the ADB procedures?** 
yes

**Are you able to click the apply button?** 
no

**Error prompted by the app, if any** 
nothing

**Comment by tribalfs** [comment]
Hi!
I'm trying to locate your device in google play store crash report but it isn't found or maybe not reflected yet.
Would you be able to take and provide a crash logcat * from your device so we can have some details of what's causing the crash on your device?

Would like you also to first  try updating your [Google Play Services](https://play.google.com/store/apps/details?id=com.google.android.gms) if you haven't have the latest installed.

*Execute the following adb command.  Do this right after the crash (device must not be rebooted yet):
`adb logcat -b crash > crashlog.txt`
_Ctrl+C_ after ~5seconds.

**Comment by chikinrais** [comment]
Thanks for your reply, I uninstalled it yesterday, and when I installed it again today and tried to see if anything would change, I found that it works fine again.
There is just one thing, it prompts me that my google play service is out of date when it starts, even though I have updated it to the latest. Strange.
Does this have anything to do with my phone being Android 11? My phone is older and doesn't support newer versions very well, so I don't plan to upgrade to 12 or above.
